### PR TITLE
Feature/Add New 'Episodes' Table

### PR DESCRIPTION
**Pull request by lukecparr**
# Description

Fixes #

- What work was done?
A new 'Episodes' table was created with no data contained in it
- Why was this work done?
This new table enables planned restructuring of existing story data to allow expanding the story library in the future
- What feature / user story is it for?
["Schema & API for Story Squad" (Trello Card)](https://trello.com/c/FM1Gd4d1)
- Any relevant links or images / screenshots
N/A

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Change Status

- [ ] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ ] Yes
- [ ] No
- [x] Not necessary

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: data/migrations/20210802234515_add-episodes-table.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+exports.up = function (knex) {
+  return knex.schema.createTable('Episodes', (t) => {
+    t.increments('ID');
```
Comment (ChadDiaz) [inline data/migrations/20210802234515_add-episodes-table.js:3]: you're using "t.increments("ID")" but in reviewing other migration files, it seems the convention is to do "episodes.increments("ID") or an abbreviation such as "ep.increments("ID")"

**Review by ChadDiaz — CHANGES_REQUESTED** [review]

File: data/migrations/20210802234515_add-episodes-table.js (hunk 1 above)
Comment (lukecparr) [reply data/migrations/20210802234515_add-episodes-table.js:3]: 't' matches conventions from many of the earlier migration files so I think this is acceptable.

**Review by ChadDiaz — APPROVED** [review]
Clean code!  Good Job!

**Review by nico-herrera — APPROVED** [review]
looks good to me!

File: data/migrations/20210802234515_add-episodes-table.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    t.integer('StoryID')
+      .notNullable()
+      .unsigned()
+      .references('StoriesNew.ID')
+      .onDelete('CASCADE');
+    t.integer('EpisodeNumber');
```
Comment (tinomen) [inline data/migrations/20210802234515_add-episodes-table.js:9]: Let's create a new migration to alter the name of this field. Currently to access the field you have to type `episodes.episodenumber` putting the table name in the field is redundant. For basic queries, we just want `episodes.number`. Advanced queries with joins can rename the field in the select clause.

Comment (lukecparr) [reply data/migrations/20210802234515_add-episodes-table.js:9]: That's a good point, we were going for clarity when looking at the db itself but your comment makes sense. 👍🏻

**Pull request merged**